src/pages/Details.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import arrow from '../assets/up-arrow.png';
+
+function Details() {
+  const { name } = useParams();
+  const [pokemon, setPokemon] = useState([]);
+
+  const fetchPokemon = async () => fetch(`https://pokeapi.co/api/v2/pokemon/${name}`)
+    .then((response) => response.json())
+    .then((data) => setPokemon(data));
```

Comment:
@mckent05 I don't understand what needs to be done because in the project requirements, it is said that the Details page should be the one to get data from the API. That data is already stored in the state so saving the data would create a duplicate.
Should I instead create an action in the reducer and then fetch that action through the Details page?

Reply:
- Yes precisely. or you can choose to create a new reducer in another file, so as to keep your project modular and make your API calls there. I hope you understand. I'm sorry for the late reply 🙏